doc/source/index.rst

```diff
@@ -34,36 +39,34 @@ Customization Guide
 -------------------
 
 JupyterHub can be configured and customized to fit a variety of deployment
-requirements. This guide helps outline how to customize and extend a
-JupyterHub deployment.
+requirements. If you'd like to know how to expand and customize your jupyterhub setup, such
```

Comment:
JupyterHub

Reply:
If you would like to expand JupyterHub, customize its setup, increase the computational resources available for users, or change authentication services, this guide will walk you through the steps. See the ...